packages/governance/src/paramManager.js

```diff
@@ -0,0 +1,119 @@
+// @ts-check
+
+import { makeStore } from '@agoric/store';
+import { assert, details as X } from '@agoric/assert';
+import { assertIsRatio } from '@agoric/zoe/src/contractSupport';
+import { AmountMath, looksLikeBrand } from '@agoric/ertp';
+/**
+ * @type {{
+ *  AMOUNT: 'amount',
+ *  ANY: 'any',
+ *  BRAND: 'brand',
+ *  INSTANCE: 'instance',
+ *  INSTALLATION: 'installation',
+ *  NAT: 'nat',
+ *  RATIO: 'ratio',
+ *  STRING: 'string',
+ * }}
+ */
+const ParamType = {
+  AMOUNT: 'amount',
+  ANY: 'any',
+  BRAND: 'brand',
+  INSTANCE: 'instance',
+  INSTALLATION: 'installation',
+  NAT: 'nat',
+  RATIO: 'ratio',
+  STRING: 'string',
+};
+harden(ParamType);
+
+const assertType = (type, value, name) => {
+  if (!type) {
```

Comment:
Just pointing to this directly because I think the comment got lost.

I think this is unnecessary since we already have the ParamType.ANY which is handled below.